app/assets/javascripts/components/forms/calendar.jsx

```diff
@@ -15,11 +15,15 @@ class Calendar extends PureComponent {
     const date = input.value ? moment(input.value) : moment().add(7, 'day');
 
     return (
-      <DateField
-        dateFormat="YYYY-MM-DD"
-        defaultValue={date.format('YYYY-MM-DD')}
-        onChange={input.onChange}
-      />
+      <div className='calendar-wrapper'>
+        <DateField
+          forceValidDate
+          dateFormat="YYYY-MM-DD"
+          defaultValue={date.format('YYYY-MM-DD')}
+          onChange={input.onChange}
+        />
+        { touched && error && <span className='input-form-error'>{error}</span> }
```

Comment:
@ricardo-quinones  strange. `touched` is always false here.

Think Calendar component is swallowing this state up? Figured Calendar `input` dom would propagate touched but it's not here.